.github/release-drafter.yml

```diff
@@ -69,4 +69,4 @@ template: |
   ## Compatibility
 
   - NGINX Ingress Controller NIC_VERSION_REPLACE_ME!
-  - Openshift 4.5 or newer.
+  - Openshift 4.6 or newer.
```

Comment:
are we deprecating an OpenShift version in a minor operator release? 🤔 
or we're just fixing the statement because the min version is already 4.6?

Reply:
just fixing the statement there